s3/pkg/utils/utils.go

```diff
@@ -134,3 +136,13 @@ func GetBackend(ctx context.Context, backedClient backend.BackendService, backen
 	backend := backendRep.Backends[0]
 	return backend, nil
 }
+
+func GetVersionId() (versionId uint64, timeStamp int64) {
+	now := time.Now().UTC().Unix()
+	version := math.MaxUint64 - uint64(now)
```

Comment:
IMHO, it's better to use nanoseconds instead of seconds, otherwise conflict may be easy to happen.

Reply:
And even no need to use math.MaxUint64 - now, but use now it self.